modules/installation-bare-metal-agent-installer-config-yaml.adoc

```diff
@@ -76,6 +76,35 @@ Class E CIDR range is reserved for a future use. To use the Class E CIDR range,
 <9> The cluster network provider Container Network Interface (CNI) plug-in to install. The supported values are `OVNKubernetes` (default value) and `OpenShiftSDN` .
 <10> The IP address pool to use for service IP addresses. You can enter only one IP address pool. This block must not overlap with existing physical networks. If you need to access the services from an external network, configure load balancers and routers to manage the traffic.
 <11> You must set the platform to `none` for a single-node cluster. You can also set the platform to `vSphere` and `baremetal`.
++
+[NOTE]
+====
+When you set the platform to `vSphere` or `baremetal`, you can configure IPv4 and IPv6 address endpoints for cluster nodes.
+
+.Example
+[source,yaml]
+----
+networking:
+  clusterNetwork:
+    - cidr: 172.21.0.0/16
+      hostPrefix: 23
+    - cidr: fd02::/48
+      hostPrefix: 64
+  machineNetwork:
+    - cidr: {{ extcidrnet }}
+    - cidr: {{ extcidrnet6 }}
+  networkType: OVNKubernetes
+  serviceNetwork:
+    - 172.22.0.0/16
+    - fd03::/112
+platform:
+  baremetal:
+    apiVIPs:
+    - 192.168.11.3
```

Comment:
Since dual stack VIPs will be supported in 4.12 recommend adding a 2nd VIP for IPv6 that matches the IPv6 machineNetwork, for both apiVIPs and ingressVIPs